### ISSUE TITLE
Update UI Tests to use Aztec accessibility identifiers for header options

### ISSUE THREAD

**Issue by rachelmcr** [wordpress-mobile/WordPress-iOS#7014]
The Aztec editor text formatting UI tests (https://github.com/wordpress-mobile/WordPress-iOS/pull/7013) currently rely on strings (such as `Heading 1`) for selecting the header style (e.g. in the `testSimpleFormatHeadingOne` test). Ideally the tests will use accessibility identifiers everywhere instead of strings, to support running the tests in any language.

Once https://github.com/wordpress-mobile/WordPress-iOS/issues/7005 is closed the tests should be updated to use accessibility identifiers for those header tests.

**Comment by rachelmcr** [comment]
Closing — will revisit this in the Aztec repo directly.
